__tests__/crawler.test.js

```diff
@@ -87,6 +87,7 @@ test("Crawl the docs and execute tests", async () => {
     `https://installer.calicocloud.io/manifests/v3.16.1-0/manifests`,
     `https://installer.calicocloud.io/manifests/v3.16.1-5/manifests`,
     `https://installer.calicocloud.io/manifests/v3.16.1-11/manifests`,
+    `https://installer.calicocloud.io/manifests/v3.17.1-11/manifests`,
```

Comment:
@tmjd is this the right manifest version for v3.17?

Reply:
This needs to be `v3.17.1-0`

Reply:
Fixed.